README.md

```diff
@@ -23,10 +23,27 @@
 1. Roadmap
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
```

Comment:
It can be used without composer too.

Reply:
właśnie, nie chcieliśmy się trochę odcinać od composera, że to tylko nasz scheduler, bo to zdanie brzmi jakbyśmy byli nadbudówką nad Composera

Reply:
@bartoszwalacik dokładnie tak jak piszesz zrozumiałem to zdanie

Reply:
check the current version

Reply:
👍